schemas/webhooks/webhook_event.yaml

```diff
@@ -0,0 +1,31 @@
+type: object
+properties:
+  payload:
+    type: object
+    required:
+      - resource_type
+      - resource_id
+      - event
+    properties:
+      resource_type:
+        type: string
+        description: Indicates the resource affected by this event.
+      action:
+        type: string
+        allOf:
+          - $ref: definitions.yaml#/webhook_event_type
+        description: The event that triggered this webhook.
+        example: report.completed
+      object:
+        type: object
+        required:
+          - id
+          - href
+        description: The events that should retrigger webhooks. Accepts values check.completed.
+        properties:
```

Comment:
Optional properties missing inside object:
- completed_at_iso8601
- status
- task_spec_id
- task_def_id
- workflow_run_id
- started_at_iso8601
- activity
- category
- created_at_iso8601
- user (object)

Reply:
I don't really know if we want to define all of them. Idea was to leave specific fields as `additionalResource`. Wdyt?

Reply:
These not defined fields can be accessed using the square brackets `[]` operator in node (and similar ways should exists in other languages). As discussed in PM, let's only define the properties at top level or most used for now.